README.md

```diff
@@ -44,6 +46,15 @@ permitted_domains: 'hashrocket.com|hshrckt.com'
 permitted_emails: 'friend@whitelist.com'
 ```
 
+##### To get admin route working on localhost:
+
+```
```

Comment:
I think that in this case making the text noticeable helps the document's readability. Would you prefer bold tags instead of fenced code?. 

I felt that this was an additional sub-section. And wanted to stay in convention with the current readme's using fencing for each section.